src/constants/icar_constants.f90

```diff
@@ -7,13 +7,25 @@ module icar_constants
     implicit none
 
     character(len=5) :: kVERSION_STRING = "2.1"
-
+    
+    ! Define team IDs for coarray teams
```

Comment:
It seems like a lot of the additions to icar_constants are related to other changes to be made in the future?  Would it be OK to only add the changes needed to implement the RK3 advection + WRF flux-correction?

Reply:
Yea, I did push all of the constant changes in this one go... my thinking was that the additional constants shouldn't change the functionality of the rest of the code, and I frankly just didn't want to manually adapt and push 5 versions of the constants file